backend/common/src/main/java/ai/verta/modeldb/common/authservice/AuthInterceptor.java

```diff
@@ -24,6 +28,24 @@
   public <R, S> Listener<R> interceptCall(
       ServerCall<R, S> call, Metadata requestHeaders, ServerCallHandler<R, S> next) {
     Context context = Context.current().withValue(METADATA_INFO, requestHeaders);
+
+    // validate empty headers from user request
+    Metadata.Key<String> email_key = Metadata.Key.of("email", Metadata.ASCII_STRING_MARSHALLER);
+    Metadata.Key<String> dev_key =
+        Metadata.Key.of("developer_key", Metadata.ASCII_STRING_MARSHALLER);
```

Comment:
We must check either `developer_key` or `developer-key` is present.

Reply:
That i will add it.